ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,55 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, node_types=('worker',)):
```

Comment:
Do you think it is a good idea to use existing get_node_resource_utilization_from_adm_top() instead of creating a new function.
something like adding a new arg "" prettytable" "
get_node_resource_utilization_from_adm_top(nodename=None, node_type='worker', PrettyTable=<boolean>)
based on the prettytable arg we call the below steps,
If true, -> print in table format
false -> skip

WDYT?

Reply:
It is good idea. 
can i change these functions ?(get_node_resource_utilization_from_adm_top, get_node_resource_utilization_from_oc_describe)

Reply:
@OdedViner feel free to change and add the extra parameter that changes behavior only for new tests.